### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by inexuscore** [driftyco/ionic-ion-drawer#28]
Hi,
Just came across this repo. haven't been able to get it to work, no README, instructions or something.
And there's 9 open PRs, any ideas on when these will be merged?

Thanks.

**Comment by iiitmahesh** [comment]
currently working on it.I have found some bugs, I need to fix it. when it was  perfect I will get back to this . Hopefully soon

**Comment by m-vdv** [comment]
@iiitmahesh Please merge and update this project. Would love to use it in my Ionic app.

PS: Is this the same code that power the drawer in Ionic 2 material design?:
http://ionicframework.com/docs/v2/components/#menus 

Because that one is really slick :)

**Comment by m-vdv** [comment]
@driftyco @mlynch Please merge and update this project.

**Comment by m-vdv** [comment]
@driftyco @mlynch anyone?

**Comment by tomloprod** [comment]
@driftyco @mlynch Yes, this seems dead...

**Comment by m-vdv** [comment]
@tomloprod Too bad, we really need a good drawer style menu for Ionic 1... 
@driftyco @mlynch is anyone could at least merge the open pull requests or please revive this project.
Thanks!

**Comment by tomloprod** [comment]
@m-vdv Yes... I do not like `ion-side-menus`  I will try to adapt this project to make it more functional (since it has a few bugs) ... When I have some free time.
